app/components/input/input-form.tsx

```diff
@@ -0,0 +1,30 @@
+import {
+  Input as ChakraFormInput,
+  InputProps as ChakraInputProps,
+  InputGroup,
+  forwardRef,
+  useColorModeValue
+} from '@chakra-ui/react'
+
+export const InputForm = forwardRef(
```

Comment:
Could you also add to storybook as this is the UI for everywhere you can use? Textarea is same!

Reply:
I created a storybook for the custom date time picker, input form, text area as well as input search! 
It's rather simple. Let me know what you think!